app/views/miq_request/_request.html.haml

```diff
@@ -116,7 +122,11 @@
               :title                 => t)
           %span#buttons_off
             = button_tag(t, :class => "btn btn-primary disabled")
-  %br
+  - if RequestLog.exists?(:resource_id => @miq_request.id)
+    %h3
+      = _("Request Logs")
+      = react('RequestsTable', {:initialData => RequestLog.where(:resource_id => @miq_request.id)})
+    %br
```

Comment:
I think this can be more cleanly written as this, but we can merge without this change.

```suggestion
  - if @miq_request.request_logs.any?
    %h3
      = _("Request Logs")
      = react('RequestsTable', {:initialData => @miq_request.request_logs})
    %br
```

Reply:
Gave this change a shot but it seems to cause this error
![image](https://user-images.githubusercontent.com/64800041/218265699-8f4ec941-445e-4de9-8010-1dda2860708b.png)

Reply:
@DavidResende0 I fixed, in the core PR, what was causing this to fail - can you try again?